src/tests/header.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+// import { render, fireEvent, cleanup } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
```

Comment:
Please kindly remove unused code for it makes your code base look untidy and can lead to you having errors